commands/stats.ts

```diff
@@ -0,0 +1,43 @@
+import { Message } from "discord.js";
+import { Command, CommandDefinition, Action } from ".";
+
+export const description: CommandDefinition = {
+  name: "Stat Hoarder",
+  description: "Displays server statistics",
+  usage: ["!stats"],
+  keys: ["stats"],
+};
+
+export const action = (message: Message) => {
+    const totalMembers= message.guild?.members.cache.size;  
+    let botCount = message.guild?.members.cache.filter(m => m.user.bot).size; 
+    if (botCount !== undefined) botCount++; // Bot doesn't include itself in the botCount lol
+
+    //Get role counts
+    let rollCounts : {rName:string; count:number}[] = [];
+
+    const rolesJSON = message.guild?.roles.cache.toJSON();
+    for (const key in rolesJSON) {
+      const roleData = rolesJSON[key]; // TODO - figure out warning 
```

Comment:
Need some help with this, not sure why I'm not allowed to do this

Reply:
I'll check it out tomorrow; done work on this for today!

Reply:
You could just do `roleData: any` if that's the problem. Maybe post the error here?

Reply:
![image](https://user-images.githubusercontent.com/43019056/104775895-216cb900-5747-11eb-842f-8e01a4ac69c0.png)

Even with `roleData: any` it throws this at the moment

Reply:
fixed